scripts/metrics/start-kar-metrics.sh

```diff
@@ -46,10 +46,9 @@ helm repo update 2>/dev/null
 echo "... Wait, installing Prometheus, Grafana, and other goodies ..."
 
 # Prometheus Server, Prometheus Operator, Grafana, Node Exporter, K8s Dashboards
-helm install prometheus prometheus-community/kube-prometheus-stack -f prom-overrides.yaml --wait -n $NAMESPACE  2>/dev/null
+helm install prometheus prometheus-community/kube-prometheus-stack -f prom-overrides.yaml --wait -n $NAMESPACE  2>/dev/null || true
```

Comment:
It's ok to continue if the install didn't work?

Reply:
That change looks bad. Removed it, and added comment about redis.